scio-core/src/main/scala/com/spotify/scio/transforms/package.scala

```diff
@@ -39,16 +39,14 @@ import scala.util.{Failure, Success, Try}
  */
 package object transforms {
   @deprecated(
-    "Class AsyncLookupDoFn was renamed to BaseAsyncLookupDoFn." +
-      "see https://spotify.github.io/scio/migrations/v0.8.0.html#async-dofns for more information",
+    "Class AsyncLookupDoFn was renamed to BaseAsyncLookupDoFn." + "see https://spotify.github.io/scio/migrations/v0.8.0.html#async-dofns for more information",
```

Comment:
So these are excluded from line width rule now? the `" + "` in the middle is now redundant.

Reply:
don't like this either I need to change this